packages/path_provider/path_provider/pubspec.yaml

```diff
@@ -2,7 +2,7 @@ name: path_provider
 description: Flutter plugin for getting commonly used locations on the Android &
   iOS file systems, such as the temp and app data directories.
 homepage: https://github.com/flutter/plugins/tree/master/packages/path_provider/path_provider
-version: 1.6.10
+version: 1.6.11
```

Comment:
My impression is that the convention is to use `+1` for a readme change, rather than incrementing the version.

@cyanglaz is that correct?

Reply:
I tried that and it caused build failures on CI, telling me to increment the version.

Reply:
if a plugin is post 1.0.0, then in x.y.z, z is the patch. so 1.6.11 would be the correct update here.
If the plugin is before 1.0.0, then it is 0.x.y+z

Reply:
Okay, thanks! I guess I'd just been seeing entirely pre-1.0 plugins.